CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+## 1.0.0
+  * Beta release changes [#23](https://github.com/singer-io/tap-iterable/pull/23)
+    * Discovery and API request fixes [#20](https://github.com/singer-io/tap-iterable/pull/20)
+      * Add api_key in header
+      * Replication key changes of `users` stream to `profileUpdatedAt` from `createdAt`
+      * Add inclusion="automatic" for replication keys in the schema
+    * Add missing fields in schema [#21](https://github.com/singer-io/tap-iterable/pull/21) 
+    * Bookmarking related changes [#17](https://github.com/singer-io/tap-iterable/pull/17)
+      * Changes in behaviour of `templates` and `campaigns` streams to work as INCREMENTAL (TDL-22206)
+      * Fix to parse the response of list_users stream properly (TDL-22213)
```

Comment:
Are these Jira ticket numbers really required here? I see they are already available in child PRs